### PR TITLE
Update all terraform providers

### PR DESCRIPTION
**Pull request by talset**
Update the aws/google provider version and write a `how to update` doc

Closes: https://github.com/cycloidio/terracognita/issues/279

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ Terracognita currently imports AWS, GCP and AzureRM cloud provider as Terraform
 Please see the following versions as follow:
 
 Providers:
- * AWS: 3.40.0
- * AzureRM: 2.58.0
+ * AWS: 4.7.0
+ * AzureRM: 2.99.0 (e1c64038e7d661d6e3a1dbfc7075f603bea9f44d)
```
Comment (xescugc) [inline README.md:27]: If it's an specific version then the vesrion is fine don't need the commit hash.

File: go.mod [hunk 2]
```diff
@@ -153,3 +172,5 @@ replace github.com/hashicorp/terraform => github.com/cycloidio/terraform v0.13.5
 
 // Fork of Azurerm that has the V2 of the SDK
 replace github.com/hashicorp/terraform-provider-azurerm => github.com/cycloidio/terraform-provider-azurerm v1.44.1-0.20220324163521-5340b36d0d63
+
+replace github.com/hashicorp/terraform-provider-aws => /opt/go/src/github.com/hashicorp/terraform-provider-aws
```
Comment (xescugc) [inline go.mod:176]: Nop.

**Review by xescugc — CHANGES_REQUESTED** [review]
Missing changelog entry.

File: go.mod (hunk 2 above)
Comment (talset) [reply go.mod:176]: don't worry, PR is still in progress, I haven't time to push the terraform-provider-aws fork yet ;)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -378,3 +378,12 @@ google_compute_instance
 #### Add a new provider
 
 For this, please open an issue to describe the provider that you want to add. We will discuss about the best way to help you in the implementation.
+
+
+#### How to update terraform-provider-google
+
+> Only if google provider need to updated
```
Comment (xescugc) [inline CONTRIBUTING.md:385]: Do we need the `>`?

Comment (talset) [reply CONTRIBUTING.md:385]: I wanted to have it as a warning but less scary. But if you prefer we could remove the `>` and keep it as a normal sentence ;)

Comment (xescugc) [reply CONTRIBUTING.md:385]: Yes I think is better, `>` normally is used to quote something from somewhere IMO hehe.

**Review by xescugc — COMMENTED** [review]
I thik we should add a comment on the ` How to update terraform-provider-aws` that only people from Cycloid will be able to do that, if not open an issue asking for it, just to have it clear.

**Comment by talset** [comment]
Doc updated for aws & azurerm

**Review by xescugc — APPROVED** [review]

**Pull request merged**